ios/Classes/AdProxy.swift

```diff
@@ -29,13 +29,13 @@ extension AdProxy {
       request.contentURL = contentUrl
     }
     if let tagForChildDirectedTreatment = options["tagForChildDirectedTreatment"] as? NSNumber {
-      request.tag(forChildDirectedTreatment: tagForChildDirectedTreatment.boolValue)
+      GADMobileAds.sharedInstance().requestConfiguration.tag(forChildDirectedTreatment: tagForChildDirectedTreatment.boolValue)
```

Comment:
We should move these options to the module as they are not on a "per request" basis anymore and instead apply to all requests. Can you check with the Android API if this is a viable solution?

Reply:
@janvennemann , I have updated the pr with required changes, for android side these changes will be done in https://jira.appcelerator.org/browse/MOD-2656